### PR TITLE
#57748: Disable the Cancel link for C&P appointments in the UI

### PR DESCRIPTION
**Pull request by simiadebowale**
This PR:
removes cancel link for c&p appointments
Screenshot 2023-05-18 at 10 03 46 PM

### CODE REVIEW

**Review by ryanshaw — APPROVED** [review]
LGTM!

File: src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/NoOnlineCancelAlert.jsx [hunk 1]
```diff
@@ -23,21 +27,34 @@ export default function NoOnlineCancelAlert({ appointment, facility }) {
       backgroundOnly
     >
       {!facility &&
+        !isCompAndPenAppointment &&
         'To reschedule or cancel this appointment, contact the VA facility where you scheduled it.'}
       {!!facility &&
+        !isCompAndPenAppointment &&
```
Comment (vbahinwillit) [inline src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/NoOnlineCancelAlert.jsx:33]: This always throws me for a loop. What's the difference between the 1st !facility and the 2nd !!facility?

**Review by vbahinwillit — COMMENTED** [review]
Just 1 question.

File: src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/NoOnlineCancelAlert.jsx (hunk 1 above)
Comment (simiadebowale) [reply src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/NoOnlineCancelAlert.jsx:33]: !facility will return true if facility data is undefined or null. 
!!facility will return false

**Pull request merged**